src/components/layouts/logged-in/container-component.vue

```diff
@@ -6,30 +6,25 @@
       </div>
 
       <div class="d-flex align-center">
-        <el-tag
-          v-if="isSuperAdmin && $store.getters.selectedAccount.status === accountsStatuses.Disabled"
-          type="danger"
-          effect="dark"
-          size="small"
-          class="mr-4"
-        >
+        <el-tag v-if="isSuperAdmin && $store.getters.selectedAccount.status === accountsStatuses.Disabled" type="danger"
+          effect="dark" size="small" class="mr-4">
           <i class="ri-tools-line" />
           {{ $t('layouts.logged-in.account-disabled') }}
         </el-tag>
 
+        <el-avatar size="medium" :src="selectedAccount.logoUrl" style="margin-right: 20px;" />
```

Comment:
Pas vraiment besoin de le mettre dans un avatar pour moi, sachant qu'il manquerait le `v-if` dans le cas où l'on n'a pas de `logoUrl`.

Petite précision aussi, t'as mis un style inline `margin-right: 20px;`. Je te laisse checker le dossier `src/styles/shared`. Il s'agit de tous les styles qui sont partagés par `maestro` à nos applications.
Dans ton cas du coup, le fichier `src/styles/shared/spaces/_margin.scss` va t'intéresser parce que du coup, pour appliquer une margin right de 20px, tu n'as cas utiliser la class `mr-5`.
Si besoin, on peut se faire un call pour que je te parle de cette partie style dont on n'a pas vraiment parlé !

```suggestion
        <img v-if="selectedAccount.logoUrl" :src="selectedAccount.logoUrl" alt="Account Logo" class="account__logo">
```

Avec la class `account__logo` que tu peux mettre dans `src/styles/theme/components/m-layout/container-component.scss`. J'ai mis ça comme style ligne 30 perso pour que ça rende pas mal :
```scss
& .account__logo {
  @extend .mr-5;
  height: 85px;
}
```